distribution-packages/test.sh

```diff
@@ -8,6 +8,9 @@ runtime_id=$(../runtime-id)
 # This might be the final/only netstandard version from now on
 netstandard_version=2.1
 
+# disabled for alpine
+[ -z "${runtime_id##alpine*}" ] && { echo Disabled for Alpine; exit 0; }
```

Comment:
We generally add the RID to `ignoredRIDs` element in the `test.json` file. That said, I am not sure [our test runner knows about `linux-musl-` yet](https://github.com/redhat-developer/dotnet-bunny/blob/main/Turkey/PlatformId.cs).

Reply:
That's what I figured, which is what prompted this. Eventually we should test using `apk`, like was done with other tests (like `man-page`, but alas I don't have the time for that, and I'd need to brush up on python.